recipes/geant4/meta.yaml

```
@@ -0,0 +1,39 @@
+{% set version = "10.5.0" %}
+
+package:
+    name: geant4
+    version: {{ version }}
+source:
+    url: http://cern.ch/geant4-data/releases/geant4.10.05.tar.gz
+    sha256: 2a86499d8327abc68456e5d7fc0303824e5704322291b331857cf4042286656e
+build:
+    number: '0'
+    skip: True  # [win]
+requirements:
+    build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - cmake
+        - make  # [unix]
+    host:
+        - expat
+        - xerces-c
+    run:
+        - expat
+        - xerces-c
+test:
+    requires:
+        - cmake
+        - make  # [unix]
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+about:
+    summary: 'Geant4 is a toolkit for the simulation of the passage of particles through matter.'
+    dev_url: https://github.com/Geant4/geant4
+    doc_url: https://geant4.web.cern.ch/support
+    home: https://geant4.web.cern.ch/
+    license: Geant4
+    license_file: LICENSE
+extra:
+    recipe-maintainers:
+        - tschoonj
```

Do you mind if I am also listed as a maintainer of this?
```suggestion
        - tschoonj
        - chrisburr
```